src/components/ColorPicker/docs/index.story.tsx

```diff
@@ -91,7 +94,8 @@ export default {
       tab({
         title: 'Usage',
         sections: [
-          description(Readme),
+          description('An implementation of a ColorPicker for TPAs'),
```

Comment:
I would try to be more informative and explain in a few words what the component lets us do. Maybe there is some description in Zeroheight?